mods/ascent/mobs/bodyparts_insectoid.dm

```diff
@@ -77,4 +77,31 @@
 
 /obj/item/organ/internal/eyes/insectoid
 	name = "compound ocelli"
-	icon_state = "eyes-compound"
\ No newline at end of file
+	icon_state = "eyes-compound"
+
+/obj/item/organ/internal/egg_sac/insectoid
```

Comment:
There is no pre-existing `/obj/item/organ/internal/egg_sac`, declaring this path with `/insectoid` creates one in the type tree but doesn't use it for anything.